app/src/main/java/com/pokemonshowdown/data/ItemDex.java

```diff
@@ -92,9 +110,10 @@ public String getItemName(String name) {
 
     public JSONObject getItemJsonObject(String name) {
         try {
-            String item = mItemDexEntries.get(MyApplication.toId(name));
+            String item = mAppContext.getString(mItemDexEntries.get(MyApplication.toId(name)));
             return new JSONObject(item);
         } catch (NullPointerException | JSONException e) {
+            e.printStackTrace();
```

Comment:
Let's not return `null`. Why don't we add a placeholder item so that future updates won't break anything? You should add this in data generator as well. Thanks!